modules/ROOT/pages/_partials/generic-obtaining-the-mobile-sdk-config-file.adoc

```diff
@@ -27,11 +27,11 @@ Locate this file at the following location in your application project:
 
 ****
 
-Locate this file at the following location in your application project:
+Move mobile-services.json to the following location in your application project:
 
 `<app directory>/mobile-services.json`
 
-NOTE: Make sure this file is visible in Xcode, add the file if necessary.
+NOTE: Ensure that mobile-services.json is visible in Xcode, add this file if necessary.
 
```

Comment:
seems weird, how about Ensure that mobile-services.json is a member of the project in the Xcode File Inspector.

Reply:
no idea. @jhellar ?

Reply:
`Ensure that mobile-services.json is a member of the project in the Xcode File Inspector.` -  LGTM. But maybe `Project navigator` instead of `File Inspector`.